src/GraphQL.Tests/DI/GraphQLBuilderTests.cs

```diff
@@ -30,12 +34,14 @@ private void Verify()
 
         private void MockSetupRegister<TService, TImplementation>(ServiceLifetime serviceLifetime = ServiceLifetime.Singleton, bool replace = false)
         {
+            _builderMock.Setup(b => b.Services).Returns(_builder).Verifiable();
```

Comment:
Let's move this into the constructor, since it's necessary by nearly every test.  If necessary, we can remove `.Verifiable()` since as long as the mocked methods on `IServiceRegister` execute, we are fine.

Reply:
(If you disagree, disregard this comment; it's fine as-is also.)

Reply:
moved